threedigrid_builder/base/settings.py

```diff
@@ -39,6 +44,10 @@ class GridSettings:
     @classmethod
     def from_dict(cls, dct):
         """Construct skipping unknown fields and None values"""
+        schema_to_builder_map = {'minimum_cell_size': 'grid_space',
+                                 'calculation_point_distance_1d': 'dist_calc_points',
+                                 'nr_grid_levels': 'kmax'}
+        dct = replace_keys(copy.copy(dct), schema_to_builder_map)
```

Comment:
In discussion with @daanvaningen we decided to not change the variable names downstream. Therefore this, ugly, mapping is needed. I am slightly worried that more of these are needed in the future creating a monstrosity. So I'm open to suggestions, and we can always fix this later on in the schema upgrade is a monster indeed appears.